package.json

```diff
@@ -1,106 +1,76 @@
 {
-  "name": "muisti-management",
-  "version": "0.1.0",
+  "name": "noheva-management",
   "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview"
+  },
   "dependencies": {
-    "@material-ui/core": "^4.11.0",
-    "@material-ui/icons": "^4.5.1",
-    "@material-ui/lab": "^4.0.0-alpha.46",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^7.1.2",
-    "@types/geojson": "^7946.0.7",
-    "@types/jest": "^24.0.0",
-    "@types/leaflet": "^1.5.12",
-    "@types/leaflet-draw": "^1.0.2",
-    "@types/node": "^12.0.0",
-    "@types/react": "^16.9.0",
-    "@types/react-beforeunload": "^2.1.0",
-    "@types/react-color": "^3.0.1",
-    "@types/react-dom": "^16.9.0",
-    "@types/react-resizable": "^1.7.2",
+    "@emotion/react": "^11.10.6",
```

Comment:
Change to strict package versioning. See oss-surveys-management for reference, 
- https://github.com/Metatavu/oss-surveys-management/blob/develop/.npmrc
- https://github.com/Metatavu/oss-surveys-management/blob/develop/package.json